ArduPlane/quadplane.cpp

```diff
@@ -608,10 +609,11 @@ bool QuadPlane::setup(void)
         }
         frame_class.set_and_save(new_value);
     }
+    gcs().send_text(MAV_SEVERITY_INFO, "QuadPlane initialise, class: %d, type: %d", frame_class.get(), frame_type.get());
     
     if (hal.util->available_memory() <
         4096 + sizeof(*motors) + sizeof(*attitude_control) + sizeof(*pos_control) + sizeof(*wp_nav) + sizeof(*ahrs_view) + sizeof(*loiter_nav)) {
-        gcs().send_text(MAV_SEVERITY_INFO, "Not enough memory for quadplane");
```

Comment:
Wouldn't all of this be vastly simplified by just jumping straight over to `AP_BoardConfig::config_error`?!

Reply:
I had forgotten about this PR.
By "vastly simplified" do you mean to eliminate the send_text calls, or just to move the error reporting into config_error?
And what is the  "add instructions" patch?

Reply:
There are two patches here - one which tells the user to fix the problem and reboot.  I like that.